src/query.rs

```diff
@@ -352,6 +395,15 @@ impl<'w, Q: Query> QueryBorrow<'w, Q> {
         unsafe { BatchedIter::new(self.meta, self.archetypes.iter(), batch_size) }
     }
 
+    /// Prepare this query for repeated execution
+    ///
+    /// A prepared query can be stored independently of the [world][World] and
+    /// is used to amortise the cost of the looking up the machting [archetypes][Archetype]
+    /// and the relevant type states therein.
```

Comment:
This explanation is a bit lower-level than I'd like to ask users to understand. How about something like:

```suggestion
    /// A prepared query can be stored independently of the [`World`] to amortize query set-up costs.
```